### DISCUSSION TITLE
Next PQCP TSC meeting - Thu 6 June 1300 UTC

### DISCUSSION THREAD

**Discussion by planetf1** [pq-code-package/tsc#61] [Announcements]
The next PQCP TSC meeting will be held on 2024-06-06 1300 UTC

Invites & addition to the PQCA calendar should follow shortly.

See also https://github.com/pq-code-package/tsc/issues/60 for a discussion on scheduling this meeting on an ongoing basis, aiming to find time(s) good for everyone

**Comment by mbbarbosa** [comment]
I will be travelling, so I will miss this meeting.
Manuel

**Comment by planetf1** [comment]
See proposed agenda here -> https://github.com/pq-code-package/tsc/pull/63
